docarray/array/storage/weaviate/backend.py

```diff
@@ -32,11 +32,18 @@ class WeaviateConfig:
     protocol: Optional[str] = field(default='http')
     name: Optional[str] = None
     serialize_config: Dict = field(default_factory=dict)
-    n_dim: Optional[int] = None  # deprecated, not used anymore since weaviate 1.10
+    # vectorIndexConfig parameters
```

Comment:
we may need to keep n_dim to guarantee backward compatibility, maybe it is clear to which parameter it should correspond?